LanguageFeatures/primary-constructors/static_processing_A09_t02.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The following errors apply to formal parameters of a declaring
+/// constructor, be it in the header or in the body. Let `p` be a formal
+/// parameter of a declaring constructor in a class, enum, or extension type
+/// declaration `D` named `C`:
+/// ...
+/// A compile-time error occurs if `p` has the modifier `covariant`, but neither
+/// `var` nor `final`.
+///
+/// @description Check that it is a compile-time error if a formal parameters of
+/// a declaring constructor has the modifier `covariant`, but neither `var` nor
+/// `final`. Test extension types.
+/// @author sgrekhov22@gmail.com
+
+// TODO (sgrekhov) Add `declaring-constructors` experimental flag
+
+extension type ET1(covariant String x) {}
```

Comment:
`final` is implicit here, which means that this is not really a test of "neither `final` nor `var`". Perhaps drop the library as a whole? I think it's already covered by other libraries where the focus is on "both `final` and `covariant`".